app/pages/YourEmployerNames.scala

```diff
@@ -18,7 +18,7 @@ package pages
 
 import play.api.libs.json.JsPath
 
-object YourEmployerName extends QuestionPage[String] {
+object YourEmployerNames extends QuestionPage[Seq[String]] {
 
   override def path: JsPath = JsPath \ toString
```

Comment:
Should the JsPath be pluralised here ?

Reply:
yes, also think this should sit in the authenticated route so moved